napari/_qt/utils.py

```diff
@@ -252,6 +253,47 @@ def combine_widgets(
     )
 
 
+def add_flash_animation(widget: QWidget):
```

Comment:
Love the functionality, but can you make `duration` and `color` parameters to the `add_flash_animation`. They can have sensible defaults, but I imagine that one day we might adjust these things/ it would be nice if they were easy to change.

I also find the flash quite intense right now. I feel a lot better with duration `300ms` and the color being `QColor(128, 128, 128, 255)`. It's still very noticeable but much easier on my eyes!

Reply:
Sounds good!

Reply:
Sure, I've modifed the function to take in duration and color parameters which should make it more general (e.g. tutorial widget in the future?)
Indeed, the flash to white was very intense so its been changed to (128, 128, 128, 128) - it's still visible but not offensive.